client/src/pages/index.tsx

```diff
@@ -294,10 +293,11 @@ const Homepage: NextPage<FeedsPageInitials> = (props): JSX.Element => {
 					<Card className="blockEdges" style={{ marginBottom: 5 }} bordered={false} title="Trending Tags">
 						{handleTrendingTagsRender()}
 					</Card>
-					<Card className="blockEdges" style={{ marginBottom: 5 }} bordered={false} title="Search Tag">
-						<Select mode="multiple" style={{ width: '100%' }} placeholder="Search..">
-						</Select>
-					</Card>
+					<TagSearchingBlock
+						tagFilter={tagFilter ? tagFilter.split(',') : []}
```

Comment:
please create a constant with `useMemo` instead, otherwise it'll compare in every render